### DISCUSSION TITLE
How to debug next.config.js file?

### DISCUSSION THREAD

**Discussion by smo043** [vercel/next.js#36054] [Help]
Hello team,

Can someone please help how to debug the next config file? I have tried starting the application in debug mode in vs code with breakpoints, and it didn't help, tried adding consoles nothing prints.

To give a little more background, with a standalone build, a few npm packages are ignored, which are required at runtime—so trying to add those packages using the webpack solution in the next config file.

Any suggestions or solutions would be highly appreciated.

**Comment by icyJoseph** [comment]
If I do this:

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  webpack(config, opts) {
    console.log(config, opts);
    return config;
  },
  experimental: {
    outputStandalone: true,
  },
};

module.exports = nextConfig;
```

I can see a both the webpack config and the options being logged. I cannot unfortunately help out with the second issue. Kinda weird, because I think Next.js tries to pack everything into the standalone build, perhaps the node_modules are not discoverable? Maybe you can read on the `nft` package to see how to make it print the things it discovers, or try using:

```
 outputFileTracingRoot: path.join(__dirname, '../../'),
```

**Reply by smo043** [reply]
Thank you, I just tried and it didn't worked. Console logs are also not printing

**Reply by icyJoseph** [reply]
Where are you looking for the console log statements? (asking because in other questions I've helped with, they were looking for logs in the browser, but the logs happen in the terminal)
